### ISSUE TITLE
Fail html[xml:lang] when not on an xhtml page

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#2015]
xml:lang only works on xhtml documents. When used on an html page, these should fail:

```html
<html xml:lang="en"></html>
```

**Comment by straker** [comment]
How does this work with the `html-xml-lang-mismatch` rule?

**Comment by jkodu** [comment]
Also, `axe.utils.isXHTML` does not seem to flag correctly given a XHTML doctype

**Comment by jkodu** [comment]
PR has been merged. Closing issue.

**Comment by padmavemulapati** [comment]
Verified with the latest axe-coconut (v4.3.0.06000), if I give within `<html>` element `xml:lang` attribute , its giving a voilation , which is expected.
But that fail message , probably like -  'xml:lang is not valid ' instead 'The <html> element does not have a lang attribute'.. @WilcoFiers , Can we take it up as an update or enhancement in this case.

![image](https://user-images.githubusercontent.com/47409579/75691731-7ab43100-5cca-11ea-9dee-662565c57333.png)

**Comment by jkodu** [comment]
Nothing to document. Closing issue.
